src/components/BookItem.js

```diff
@@ -2,24 +2,54 @@ import React from 'react';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 import { removeBook, deleteBook } from '../redux/books/booksSlice';
+import '../styles/books.css';
 
 const BookItem = ({ itemProp }) => {
   const dispatch = useDispatch();
 
   return (
     <>
-      <li>{itemProp.title}</li>
-      <li>{itemProp.author}</li>
+      <li className="book-cards">
+        <div className="book-info">
+          <p className="category">{itemProp.category}</p>
```

Comment:
Greate work done so far, I would just mention that the category is missing in you cards , you can add a static values to make your design much with the template , also please reduce the vertical space between the titles and buttons refer to the images below 

![image (19)](https://github.com/ragangithub/bookstore-react/assets/59707859/89f9adea-f59a-4e00-ae74-53de9a7e6c11)
![image (24)](https://github.com/ragangithub/bookstore-react/assets/59707859/b83a6593-a65b-4a14-9157-6153fa78616e)